lib/mqtt_ascoltatore.js

```diff
@@ -84,6 +87,24 @@ MQTTAscoltatore.prototype._startConn = function() {
   return this._client;
 };
 
+MQTTAscoltatore.prototype.reconnectTopics = function reconnectTopics(cb) {
+  var that = this;
+  
+  var subscribedTopics = that._subs_counter.keys();
+  
+  var opts = {
+    qos: 1
+  };
+  
+  subscribedTopics.forEach(function(topic) {
+    that._client.subscribe(topic, opts, function() {
+      debug("re-registered subscriber for topic " + topic);
```

Comment:
I think you should use `async.parallel` here, and call `cb()` when all resubscribe finishes.